app/main.py

```diff
@@ -1,8 +1,46 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, list_cars: list) -> float:
+        income = 0
+        for car in list_cars:
+            if car.clean_mark < self.clean_power:
+                income += self.calculate_washing_price(car)
+                car.clean_mark = self.clean_power
+                self.wash_single_car(car)
+        return income
+
+    def wash_single_car(self, single_car: Car) -> None:
```

Comment:
You can name it `car` to make more correctly. Because in this case _single_ means that the method washes only one car by the time, not some specific single cars (sounds like `single car` is another type of car)